src/phase.cs

```diff
@@ -0,0 +1,1299 @@
+		InvalidInstance = 1346925667,
+		APIMisuse = 1346925668,
+		SystemNotInitialized = 1346925669,
+		OutOfMemory = 1346925670,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	[ErrorDomain ("PHASEAssetErrorDomain")]
+	public enum PhaseAssetError : long {
+		FailedToLoad = 1346920801,
+		InvalidEngineInstance = 1346920802,
+		BadParameters = 1346920803,
+		AlreadyExists = 1346920804,
+		GeneralError = 1346920805,
+		MemoryAllocation = 1346920806,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseSoundEventPrepareHandlerReason : long {
+		Error = 0,
+		Prepared = 1,
+		Terminated = 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseSoundEventStartHandlerReason : long {
+		Error = 0,
+		FinishedPlaying = 1,
+		Terminated = 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseSoundEventSeekHandlerReason : long {
+		Error = 0,
+		ErrorSeekAlreadyInProgress = 1,
+		SeekSuccessful = 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseSoundEventPrepareState : long {
+		NotStarted = 0,
+		InProgress = 1,
+		Prepared = 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseAssetType : long {
+		Resident = 0,
+		Streamed = 1,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseCurveType : long {
+		Linear = 1668435054,
+		Squared = 1668436849,
+		InverseSquared = 1668434257,
+		Cubed = 1668432757,
+		InverseCubed = 1668434243,
+		Sine = 1668436846,
+		InverseSine = 1668434259,
+		Sigmoid = 1668436839,
+		InverseSigmoid = 1668434247,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseCullOption : long {
+		Terminate = 0,
+		SleepWakeAtZero = 1,
+		SleepWakeAtRandomOffset = 2,
+		SleepWakeAtRealtimeOffset = 3,
+		DoNotCull = 4,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhasePlaybackMode : long {
+		OneShot = 0,
+		Looping = 1,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseNormalizationMode : long {
+		None = 0,
+		Dynamic = 1,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhaseCalibrationMode : long {
+		None = 0,
+		RelativeSpl = 1,
+		AbsoluteSpl = 2,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Flags]
+	[Native]
+	public enum PhasePushStreamBufferOptions : ulong {
+		Default = 1uL << 0,
+		Loops = 1uL << 1,
+		Interrupts = 1uL << 2,
+		InterruptsAtLoop = 1uL << 3,
+	}
+
+	[Mac (12,0), NoWatch, TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Native]
+	public enum PhasePushStreamCompletionCallbackType : long {
+		PushStreamCompletionDataRendered = 0,
```

Comment:
Wondering why the enum name and element are these instead of 
`PhasePushStreamCompletion` and `DataRendered = 0,`
Is it because the enum name would end with "Completion" and we want to avoid that?

Reply:
Better to have a short name, +1
```suggestion
		DataRendered = 0,
```